cpu/nrf5x_common/periph/Makefile

```diff
@@ -1,3 +1,17 @@
 MODULE = nrf5x_common_periph
 
+# Select the specific implementation for `periph_i2c`
+ifneq (,$(filter periph_i2c,$(USEMODULE)))
+  ifneq (,$(filter $(CPU_FAM),nrf52 nrf9160))
```

Comment:
so the new implementation is for *both* nrf52 and nrf9160?
What happens with nrf51?

Reply:
nRF51 still uses its own implementation (cpu/nrf51/periph).
I think @bergzand tried to unify SPI and I2C on nRF5x but gave up because it doesn't worth the effort.
But since nRF9160 has almost the same IP, better use the same driver.

Reply:
ah because this was previously in `cpu/nrf52`.
I suppose the remaining drivers there will be the same for nrf9160 as well

Reply:
> because this was previously in cpu/nrf52


Yes I simply move SPI and I2C driver to nrf5x_common folder and change doxygen group.

> I suppose the remaining drivers there will be the same for nrf9160 as well


I didn't check for now. But if this is the case, I'll add them later in followup PR ,if I can test them on my custom board. I don't think I'll be able to test ADC for example.